src/main/java/NestedNullRefactoring.java

```diff
@@ -66,15 +62,14 @@ public boolean isApplicable(MethodInvocation invocation) {
 	}
 
 	/*
-	 * Returns true iff Node is a one-line method that returns the result of a null
-	 * check
+	 * Returns true iff Node is a one-line private or package-private method that
```

Comment:
Why is package-private safe? A subclass that redefines the method could be in the same package.